README.md

```diff
@@ -1,18 +1,15 @@
 <p align="center">
-    <img src="docs/assets/brand/logo-lockup-vertical/logo-lockup-vertical.png" width=30%>
```

Comment:
@pavithraes We need absolute URLs here for this to be rendered correctly on PyPI.

- bad: https://test.pypi.org/project/Ragna/0.1.0.post0/
- good: https://test.pypi.org/project/Ragna/0.1.0rc2/

(Don't worry, this is just the PyPI test mirror)